ddtrace/contrib/wsgi/wsgi.py

```diff
@@ -165,29 +165,25 @@ def __call__(self, environ, start_response):
         headers = get_request_headers(environ)
         closing_iterator = ()
         not_blocked = True
-        with _asm_request_context.asm_request_context_manager(
-            environ.get("REMOTE_ADDR"), headers, headers_case_sensitive=True
-        ):
+        with _asm_request_context.asm_request_context_manager(environ.get("REMOTE_ADDR"), headers, True):
             req_span = self.tracer.trace(
                 self._request_span_name,
                 service=trace_utils.int_service(self._pin, self._config),
                 span_type=SpanTypes.WEB,
             )
 
             if self.tracer._appsec_enabled:
-                # [IP Blocking]
```

Comment:
Why remove the comments?

Reply:
They don't add any information beyond what I can learn from reading the code directly below them. That's a subjective call.